src/Stan.hs

```diff
@@ -34,8 +41,8 @@ run = runStanCli >>= \case
 runStan :: StanArgs -> IO ()
 runStan StanArgs{..} = do
     hieFiles <- readHieFiles stanArgsHiedir
-    -- create extensions map
-    extensionsMap <- case stanArgsCabalFilePath of
+    -- create cabal default extensions map
+    cabalExtensionsMap <- case stanArgsCabalFilePath of
```

Comment:
Would it make sense to move all code related to creating `extensionsMap` into a separate function, so the main function will look simpler? Like:

```haskell
runStan StanArgs{..} = do
    hieFiles <- readHieFiles
    cabaExtensionsMap <- createCabalExtensionsMap stanArgsCabalFilePath hieFiles
    let analysis = ...
    putTextLn ...
```

It's not necessary, but I wonder whether it will improve code readability :thinking: